hal/src/main/native/sim/HAL.cpp

```diff
@@ -11,6 +11,7 @@
 #include <wpi/spinlock.h>
 
 #ifdef _WIN32
+#undef NOMINMAX
```

Comment:
The MSVC default toolchain for bazel cannot turn this rule off. There are alternatives available, but his was the only file I found that this actually mattered.